x-pack/legacy/plugins/siem/public/pages/detection_engine/rules/components/step_rule_actions/schema.tsx

```diff
@@ -11,7 +11,7 @@ import { FormSchema } from '../../../../../shared_imports';
 export const schema: FormSchema = {
   actions: {},
   enabled: {},
-  kibanaSiemAppUrl: {},
+  kibana_siem_app_url: {},
```

Comment:
This should match the `path` of the form, so I think this should be camel cased as well.